### PR TITLE
Simple graylog integration 

### PR DESCRIPTION
**Pull request by laf**
The config options will now be shown as crossed out but still visable for those who want to set things up in config.php but has a message at the start to direct people to use the WebUI.

Docs also for setting up graylog integration.

Widget added as well for graylog data.

### CODE REVIEW

**Comment by f0o** [comment]
Schema-file collides, please bump it to 063 :)

**Comment by laf** [comment]
Done

**Comment by f0o** [comment]
Got it in my install and have a couple of notes:
 - My graylog, by default, didnt provide FQDN. I had to add force rdns in the input/s to obtain it. So i was unable to seqrch by device until that change. Should be doc'd.
- although being able to filter logs after above change, I cant click on the hostname to link me to the devices page, i'd appreciate this link.
- expand search! Could we just passthrough a searchbox that accepts same syntax as graylog?

Generally this PR is already mergeworthy. It does a good job giving an overview of the last N logmessages. Ofc it doesnt replace graylog.

**Comment by laf** [comment]
I've updated to use ip from the hostname, this means it has to resolve the correct IP for the hostname of the device.

Anything more than this is going to be difficult as we can't do a lookup for details in the database otherwise we will be performing an sql query for every log entry.

I've also made the source clickable, however this will only work if it's hostnames as we have no way to get to a device via IP and we can't get the ID without doing an sql query on all the log entries.

The search can be expanded, my testing is limited due to having to do this all on my laptop so best getting this out there for others to contribute to.

ofc it's not supposed to replace graylog, it's simple integration so you can grab the logs directly for a device or view the graylog logs. It can be expanded further so hopefully this is just the start and others can contribute to improve.

**Comment by f0o** [comment]
:+1:

**Comment by laf** [comment]
Source is now clickable.

File: html/includes/table/graylog.inc.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                     "Accept: application/json",
+    )
+));
+
+$messages = json_decode(file_get_contents($graylog_url, false, $context),true);
+
+foreach ($messages['messages'] as $message) {
+    $response[] = array(
+                      'timestamp' => $message['message']['timestamp'],
+                      'source'    => generate_url(array('page'=>'device', 'device'=>$message['message']['source'])),
```
Comment (f0o) [inline html/includes/table/graylog.inc.php]: Please set the source to:
`'<a href="'.generate_url(array('page'=>'device', 'device'=>$message['message']['source']))'">'.$message['message']['source'].'</a>'`

**Pull request merged**